sr_robot_launch/src/sr_robot_launch/sr_ur_arm_unlock.py

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env python3
+# Copyright 2020, 2021 Shadow Robot Company Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import
+import rospy
+from std_msgs.msg import Bool
+from ur_dashboard_msgs.srv import GetSafetyMode, GetProgramState, GetRobotMode, Load, IsProgramRunning
+from ur_dashboard_msgs.msg import SafetyMode, ProgramState, RobotMode
+from std_srvs.srv import Trigger
+
+
+class RobotSafetyMonitor(object):
+    def __init__(self, name):
+        self.name = name
+        self.substring = None
+        self.subscriber = None
+        self.estop_pressed = False
+        self.mode = None
```

Comment:
Maybe would be better to set those as private, and substring isn't self-explanatory

Reply:
agreed, private params, leading underscore, pass to constructor instead of filling them in manually

Reply:
71d2bc6d3854b4816d8790a2a364fe157085ec51

Reply:
Fine for me